### DISCUSSION TITLE
Draw a marker with a tooltip already bind to it

### DISCUSSION THREAD

**Discussion by cubecleveland** [geoman-io/leaflet-geoman#1315] [Questions & Help] [answered]
Hello, im using the following code to draw marker on the map on demand
```
 mapRef.current.pm.enableDraw('Marker', {
              markerEditable: true,
              markerStyle: {
                icon: getDoorIcon({ name }),
                properties: { ...e.value.properties, hidden: null }
              },
              tooltips: false,
              continueDrawing: false
            });
```
is there any way to have the marker already have a tooltip binded to it so i can see it moving on the screen with a tooltip already attached ?

**Comment by Falke-Design** [comment]
No this is not possible. You can try to catch the `pm:drawstart` event and then bind a own Tooltip to the Marker Layer over `map.pm.Draw.Marker._layer.bindTooltip('dsdsdf')`

**Comment by Falke-Design** [comment, marked as answer]
Also checkout #1242
